src/Microsoft.DotNet.Interactive.HttpRequest/HttpRequestKernelExtension.cs

```diff
@@ -16,7 +16,12 @@ public static void Load(Kernel kernel, HttpClient? httpClient = null)
             compositeKernel.Add(httpRequestKernel);
             httpRequestKernel.UseValueSharing();
 
-            RegisterFormatters();
+            // Include html, plain text and json formatted outputs in display event responses.
+            Formatter.SetPreferredMimeTypesFor(

```

Comment:
This can be done via the `TypeFormatterSourceAttribute` now.

Reply:
Ah indeed - let me change that.